### ISSUE TITLE
#2 UI implementation did not meet the minimum requirement

### ISSUE THREAD

**Issue by sleepylee** [kyawlinnthant/nimble-code-test#4]
This is quoted from our original requirement:

![image](https://github.com/kyawlinnthant/nimble-code-test/assets/13435717/06613927-d392-49bf-91a3-f7c38edfcbc7)

So technically, we would expect to see your Login and Home screen implementations that follow our provided design (but NOT the other screen i.e., forgot password, survey details).

**Comment by sleepylee** [comment]
Also to this point, I think you might have been overthinking about the login form where you set a password (for theLogin form) to be vetted with strict validator rules. I believe this should be done only for the Registration flow, right?

Another point is that you are putting password fields in plaintext, which is an awkward (but basic) UI implementation IMO:
<img src="https://github.com/kyawlinnthant/nimble-code-test/assets/13435717/b03371d6-fe57-478d-ab9b-ba0720c6fe24" width=300 />

**Comment by kyawlinnthant** [comment]
For this issue, I will 

- clear unnecessary screens
- clear unnecessary password validation rules
- Refactor TextInputLayout for password to appropriate password field ( not plaintext )

**Comment by kyawlinnthant** [comment]
Fix the issue, making PR with` issue2` branch and merge it. Commit  8666f9c0a0ae7eb660e1c64484304d78603a3bf0

**Comment by sleepylee** [comment]
hi @kyawlinnthant, just a friendly reminder that we would like to see the implementation of the Login & Home screens to **follow the design** as we provided ([in Figma](https://www.figma.com/proto/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=31%3A195&scaling=scale-down)), so you will need to make sure that you implementation matches the design. For the details of the design, you can turn on the Dev/Editor mode (instead of prototype), let us know if you can access it (or not) 😉

**Comment by kyawlinnthant** [comment]
Oh @sleepylee, I will access editor mode and make sure the font, color, and design match.

**Comment by kyawlinnthant** [comment]
Can I get access to the font, please? Can you provide me with some resources to get that font?

**Comment by sleepylee** [comment]
hi @kyawlinnthant, you can use your default font for your convenience (this is not required in specific from the requirement so you can dismiss it 😉 ).
In case you want to make it as close as possible, you can search for Neuzeit font (free) as per design.

![image](https://github.com/kyawlinnthant/nimble-code-test/assets/13435717/a8ec5ebf-aaf8-4c4c-ba1f-b46be7a30c8a)

**Comment by kyawlinnthant** [comment]
Hi @sleepylee , I just refactor my Login and Home Screens.
I added

-  Some more fields in database
- DateTimeFormatter to convert desired Date Format
- Unit Test for DateTimeFormatter
- Refactor Typo (lineSpacing, FontWeight, FontSize,...)
- Background Images and Scrim layer in Screens

<img width="622" alt="Screenshot 2023-11-09 at 11 54 48 PM" src="https://github.com/kyawlinnthant/nimble-code-test/assets/24668175/b8f5d469-f7ae-4a2d-9573-46ed3818f860">

<img width="397" alt="Screenshot 2023-11-10 at 12 26 09 AM" src="https://github.com/kyawlinnthant/nimble-code-test/assets/24668175/6856e657-4bd4-49de-b5c1-27eaf29b977c">


But for the details about design, the design don't have light mode and I just implemented with Light and Dark modes. So I need more discussions about it. And for the Surveys screen, the placement of Page Indicator has to be discussed more with designers. 

Thank you. 

Merge the changes with commit 740a191206a5ed4c4a080a88711cb67136a77ead
